command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@julz Since the primary interface to customers (for a web interface) would not be the testing shim. I argue the customer facing interface is the one that needs to be tested for compliance purposes not some shim.  A shim could have any number of hacks in it to force compliance as shown already in the history of this PR.  Yes, that means we would have to have a runtime-spec for web services or what not if such a need arose.   

@wking you need to PR the runtime-spec, and get it approved, if you want to change the runtime-spec. Changing an older parent spec from a newer child document.. probably not a good idea. And yes I'm saying you would need different specs for the different types of APIs.  And yes while we could have some common code in the testing apparatus there would be new issues to test.